src/bedtools/bedtools_bedtobam/script.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
```

Comment:
Bash will continue on errors by default. So we need to make sure that error are correctly reported
```suggestion
set -eo pipefail
```